app/src/main/java/com/example/hoaht/myapplication/MainActivity.java

```diff
@@ -10,4 +11,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+
+    private void Test() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions